### ISSUE TITLE
ReferenceError: _FileSystem_UnknownFileSystemError is not defined

### ISSUE THREAD

**Issue by joeybright** [gren-lang/node#9]
When a `FileSystem.UnknownFileSystemError` is triggered in my Gren program, it crashes with the error `ReferenceError: _FileSystem_UnknownFileSystemError is not defined`.

To reproduce, here's a minimal `Main.gren` and `gren.json`:

```elm
module Main exposing ( main )

{-|-}

import FileSystem
import Init
import Node
import Node exposing ( Environment, Program )
import Task exposing ( Task )


{-|-}
main : Program Model Msg
main =
    Node.defineProgram
        { init = init
        , update =
            case msg of
                NoOp ->
                    { model = model
                    , command = Cmd.none
                    }
        , subscriptions = \_ -> Sub.none
        }


{-|-}
type alias Model =
    {}


{-|-}
type Msg
    = NoOp


{-|-}
init :
    Environment
    -> Init.Task
        { model : Model
        , command : Cmd Msg
        }
init env =
    Init.await
        FileSystem.initialize
        (\fileSystemPermission ->
            Node.startProgram
                { model = {}
                , command =
                    FileSystem.openForRead fileSystemPermission "example"
                        |> Task.mapError (\_ -> FileSystem.UnknownFileSystemError "")
                        |> Task.andThen FileSystem.read
                        |> Task.attempt (Result.map (\_ -> NoOp) >> Result.withDefault NoOp)
                }
        )

```

```json
{
    "type": "application",
    "platform": "node",
    "source-directories": [
        "src"
    ],
    "gren-version": "0.3.0",
    "dependencies": {
        "direct": {
            "gren-lang/node": "3.0.0",
            "gren-lang/core": "4.0.1"
        },
        "indirect": {
            "gren-lang/url": "3.0.0"
        }
    }
}
```

To trigger an unknown file error in the code above, I try to read a directory entitled `example` instead of a file. It's expected for this operation to fail, but not the program to crash. If the directory doesn't exist, the program does not crash.

I'm assuming any program will crash when a `FileSystem.UnknownFileSystemError` is triggered by other means, but this is how I initially discovered it!

**Comment by robinheghan** [comment]
Thank you for reporting this issue, @joeybright , and for providing a minimally reproducible example!

I believe the issues is fixed on the `main` branch now, could you verify that the issue is fixed on your end?

If you haven't done this before, you need to clone the repo, then refer to its on-disk position in your `gren.json` file. It should look something like this:

```json
{
    "type": "application",
    "platform": "node",
    "source-directories": [
        "src"
    ],
    "gren-version": "0.3.0",
    "dependencies": {
        "direct": {
            "gren-lang/node": "local:../path/to/cloned/node/repo",
            "gren-lang/core": "4.0.1"
        },
        "indirect": {
            "gren-lang/url": "3.0.0"
        }
    }
}
```

**Comment by joeybright** [comment]
@robinheghan Thanks for the super quick fix! It's now working for me locally after cloning the `main` branch and updating my `gren.json`!

**Comment by robinheghan** [comment]
Fixed in 3.0.1
